app/controllers/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -78,6 +78,15 @@ def reassign_hostgroup
         end
       end
 
+      api :PUT, "/hosts/bulk/disassociate", N_("Disassociate hosts")
```

Comment:
Thinking more: is this a good moment to consider the naming? I know it was already there, but perhaps this is clearer what it actually does:
```suggestion
      api :PUT, "/hosts/bulk/disassociate", N_("Disassociate compute resources")
```

My reasoning is that the hosts part doesn't add anything because we're already on the hosts overview. The new subject makes it clear what will happen.

Reply:
Does this change apply only here? Or all places where `Disassociate hosts` is displayed?

Reply:
Looks like there is only one place now:
https://github.com/theforeman/foreman/blob/9ff8631bcd055e3bf7c9da49a92919abca572f73/app/helpers/host_description_helper.rb#L18

I lean to not updating the old texts because I'm not sure where else that's referred to.

Reply:
Thanks for looking into this @ekohl :+1: 

I think this task really means `disassociate hosts from their compute resources`. 
If that is correct, then `disassociate hosts` makes more sense than `disassociate compute resources`.

Reply:
> I lean to not updating the old texts because I'm not sure where else that's referred to.

I mean `Disassociate hosts` in the action menu and the modal. 
But `Disassociate hosts`  seems right to me at this moment.

Reply:
My point is that you're on the hosts page, so that bit should be implicit. But disassociate from what? Disassociate from compute resources is the answer. I left out the word from for brevity.

Perhaps Maria can weigh in on the UX aspect of it. Or @jeremylenz cpuld be a tie breaker

Reply:
`api :PUT, "/hosts/bulk/disassociate"` implies that hosts are already involved, so IMO "Disassociate compute resources" is sufficient. It could be short for any number of things ('disassociate compute resources from their hosts?' 'disassociate [hosts] from compute resources?') but we don't need all of those words.

Reply:
Updated.